sentry/src/lib.rs

```diff
@@ -103,6 +106,10 @@
 //!   dependencies.
 //! - `curl`: Enables the curl transport.
 //! - `surf`: Enables the surf transport.
+//! - `ureq`: Enables the ureq transport.
+//! - `ureq-tls`: Enables `rustls` support for `ureq`. Please note that this has to be enabled for
+//!    https support, and cannot be enabled by default.
+//! - `ureq-native-tls`: Enables `native-tls` support for `ureq`.
```

Comment:
you could recommend here to prefer setting one of `ureq-tls` or `ureq-native-tls` and mention they automatically pull in `ureq`.

Or maybe even go further and rename `ureq` to `ureq_base_` or something and make it internal only.

Reply:
That sounds like a better plan, I don't think a Sentry transport without https makes a lot of sense :sweat_smile:

Reply:
@flub I opted to go for `ureq` and `ureq-native-tls`. No `ureq_base_` though, as there's only `httpdate` support here after we get rid of `tokio`.